### PR TITLE
CSTAR-463 - fixed duplicate consistency-level name getting code

### PR DESCRIPTION
**Pull request by c-kodman**

### CODE REVIEW

**Comment by mambocab** [comment]
Code looks good to me so far. A couple things:

- Have you run this to confirm it doesn't break anything?
- Have you checked around in other files to see if there are other instances of this logic?
- You should be able to delete `TestHelper._name` now.

**Comment by mambocab** [comment]
Looks like each instance of `ConsistencyLevel.consistency_value_to_name` should be `cassandra.consistency_value_to_name`: https://github.com/datastax/python-driver/blob/master/cassandra/__init__.py#L124

**Comment by ptnapoleon** [comment]
His code is now failing, and I genuinely can't figure out why. It doesn't appear related to his changes.

File: consistency_test.py [hunk 1]
```diff
@@ -147,23 +147,23 @@ def insert_user(self, session, userid, age, consistency, serial_consistency=None
...
+        text = "UPDATE users SET age = {} WHERE userid = {}".format(age, userid)
         if serial_consistency and prev_age:
-            text = text + " IF age = %d" % (prev_age)
+            text = text + " IF age = {}".format(prev_age)
         statement = SimpleStatement(text, consistency_level=consistency, serial_consistency_level=serial_consistency)
         session.execute(statement)
 
     def delete_user(self, session, userid, consistency):
-        statement = SimpleStatement("DELETE FROM users where userid = %d" % (userid,), consistency_level=consistency)
+        statement = SimpleStatement("DELETE FROM users where userid = {}".format(userid,), consistency_level=consistency)
```
Comment (ptnapoleon) [inline consistency_test.py]: Every instance where you have `.format(variable,)` needs to be `.format(variable)`

**Comment by mambocab** [comment]
> His code is now failing

Are you talking about the log errors?

**Comment by ptnapoleon** [comment]
Ugh, the job wasn't linked on this PR so I don't remember at all what was wrong. Hopefully he can find it

**Comment by c-kodman** [comment]
http://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/15/

**Comment by mambocab** [comment]
Could it have been flakiness from the recent problems with Netty?

This needs to be rebased.

**Comment by mambocab** [comment]
But, after rebase and another run on CassCI to make sure it works, I'm +1, code looks great.

**Comment by c-kodman** [comment]
http://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/17/
I rebased with Russ's help.  I'm not positive I did it properly.  But I think I did?

**Comment by c-kodman** [comment]
actually this is the correct build:
http://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/18/

**Comment by c-kodman** [comment]
Looks like I broke this.  I'm not sure how to fix it right now either.

**Comment by c-kodman** [comment]
Okay, so I think I broke it during my rebase.  I saved my original branch in another branch of mine - dteststress_save.  If I have to go back, it is there.  Someone will just have to let me know.

**Comment by c-kodman** [comment]
https://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/19/

**Comment by c-kodman** [comment]
https://cassci.datastax.com/view/Dev/view/c-kodman/job/c-kodman-trunk-dtest/20/

**Comment by c-kodman** [comment]
The build passed (2 failures unrelated to the consistency_test).  So as far as I know, we are good to go.

**Pull request merged**